examples/viewer.cpp

```diff
@@ -135,7 +150,7 @@ bool Viewer::render()
     // wipe the drawing surface clear
     glClear(GL_COLOR_BUFFER_BIT | GL_DEPTH_BUFFER_BIT);
 
-    GLint x = 0, y = 0, width = 600, height = 400;
```

Comment:
Essentially the viewport being rendered is set to be a hardcoded dimension versus one relative to the actual window size. The result prior to the fix is all 4 frames were rendering in just one section of the window as opposed to filling the window. Then when it was filling the window, resizing the window smaller would clip the render because it wasn't dynamically changing the viewports.